components/navigation/navChild.module.css

```diff
@@ -17,6 +17,11 @@
 
 .ExternalIcon {
   @apply relative z-10 cursor-pointer text-sm ml-1;
+  @apply text-gray-70 !important;
```

Comment:
Should we be changing this color to something lighter for dark mode, or is contrast good enough?

Reply:
I picked a medium grey that I thought looked good in both dark and light mode, but happy to follow up with color tweaks. I even thought about using the category color for dividers at the second level and then grey at the third (innermost level) so there could be some distinction if a category had dividers as both sibling and child. (It looks okay in all grey, so happy for it to be a follow up unless you think the visibility is subpar as is.

Instead of forcing the color, I could increase transparency to force it to simply be a shade less pronounced in both light and dark mode. Just LMK your preference.

Reply:
After playing with it for a bit, I think the text color works well! I did update the divider lines [here](https://github.com/streamlit/docs/pull/845/commits/2df3a8d0434e18548b33d5b77719877784d8d70c), to make them less harsh. Let me know what you think!

Reply:
LGTM